### PR TITLE
docs(lambda): fromAsset exclude

### PR DESCRIPTION
**Pull request by go-to-k**
We discussed for the need to document `exclude` patterns using a negation in [the PR](https://github.com/aws/aws-cdk/pull/26365#issuecomment-1646085090).

I also could not find documentation for the `exclude` property itself, so I added them.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by tmokmss** [comment]
The sample code in doc must be compilable :) https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md#documentation

Perhaps you have to change `default.tx-fixture` to reference `IgnoreMode` in the code.

**Comment by go-to-k** [comment]
@tmokmss 

Thank you! I've never touched the rosetta file before so that helped!

I fixed it and pushed it.

**Review by colifran — DISMISSED** [review]
@go-to-k Thanks for getting started on this! Let me know what you think about making a slight wording change.

File: packages/aws-cdk-lib/aws-lambda/README.md [hunk 1]
```diff
@@ -1134,3 +1134,32 @@ new lambda.Function(this, 'Lambda', {
   code: lambda.Code.fromAsset(path.join(__dirname, 'lambda-handler')),
 });
 ```
+
+## Exclude Patterns for Assets
+
+When using `lambda.Code.fromAsset(path)` an `exclude` property allows you to ignore particular files for assets based on simple glob patterns.
+
+An `ignoreMode` property can be used to write based on the [.gitignore specification](https://git-scm.com/docs/gitignore) and the [.dockerignore specification](https://docs.docker.com/engine/reference/builder/#dockerignore-file).
```
[the remark this answers is not in the document]
Comment (go-to-k) [reply packages/aws-cdk-lib/aws-lambda/README.md:1142]: @colifran 

Thanks for your review! I think that's good, I change.

**Review by colifran — DISMISSED** [review]
@go-to-k looks good! Thank you for your contribution.

**Review by colifran — APPROVED** [review]

**Pull request merged**